lib/bot/issue_manager.rb

```diff
@@ -236,12 +240,26 @@ def get_credentials
     end
   end
 
-  def load_yaml_file
+  def send_notification(label)
     begin
-      @timestamps = YAML.load_file(ISSUE_MANAGER_YAML_FILE)
+      if @notify.has_key?(label)
+        if @notify[label].has_key?('mailto')
+          @notify[label]['mailto'].each do |address|
+            UserNotifier.send_notification_email(address, issue.number, label).deliver
+          end
+        end
+      end
+    rescue Exception => msg
+      logger.error("Couldn't send email notification. Exception: #{msg}")
+    end
+  end
+
+  def load_yaml_file(file)
```

Comment:
Not needed? because of the rails config? 
I guess we still need to call the send_notification_email